data/devicemaps/maps.json

```diff
@@ -90,7 +90,7 @@
         "locale": "en_US",
         "scancode": null
     },
-    "Razer Huntsman Elite (DE)": {
+    "Razer Huntsman Elite (Deutsche)": {
```

Comment:
Wouldn't `German` be better as the default language of the program is English? If you choose to stay with German name it `Deutsch`, `Deutsches Layout` or `Deutsche Tastaturbelegung`.

Reply:
It could, but since the label isn't translatable, I'd prefer to use the native word for "German". `Deutsch` it shall be, thanks!

`Deutsche` was from a previously contributed German layout - is this one incorrect too?

> Razer Blade Pro 2019 (Deutsche)